client/apps/admin-portal/src/app/sermon-view/sermon-view.component.ts

```diff
@@ -79,16 +70,12 @@ export class SermonViewComponent implements OnInit {
     }
   }
 
-  downloadVideo() {
-    return this.notifyService.openSnackBar('The sermon is downloading!');
-  }
-
-  listenToAudio() {
-    return this.notifyService.openSnackBar('The sermon audio is playing!');
+  handleMediaAction(mediaItemUrl: string) {
+    window.open(mediaItemUrl, '_blank');
   }
 
-  downloadNotes() {
-    return this.notifyService.openSnackBar('The sermon notes are downloading!');
+  goBack() {
```

Comment:
We don't use this back arrow anymore, but it seems like Garv wants it back.
Can you add the HTML back in, please?